swan-lake/integration/get-started.md

```
@@ -191,14 +201,26 @@ service / on new http:Listener(8080) {
+```bash
 $ curl http://localhost:8080/countries
 [{"name":"United States", "continent":"North America", "gdpPerCapita":82657.95}, {"name":"Russia", "continent":"Europe/Asia", "gdpPerCapita":15047.85}, {"name":"Mexico", "continent":"North America", "gdpPerCapita":13883.21}, {"name":"China", "continent":"Asia", "gdpPerCapita":12318.28}, {"name":"Brazil", "continent":"South America", "gdpPerCapita":9147.56}, {"name":"Indonesia", "continent":"Asia", "gdpPerCapita":4091.05}, {"name":"Egypt", "continent":"Africa", "gdpPerCapita":3547.19}, {"name":"India", "continent":"Asia", "gdpPerCapita":2318.83}]
+
 ```
 
 ## Learn more
 
-In this guide, you set up your development environment and wrote two Ballerina programs. For more learning resources, see [Learn](/learn/).
+In this guide, you set up your development environment and wrote two Ballerina programs. For more learning resources, see [Learn](https://www.google.com/search?q=/learn/).
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)